src/coreclr/jit/likelyclass.cpp

```diff
@@ -104,18 +106,19 @@ LikelyClassHistogram::LikelyClassHistogram(INT_PTR* histogramEntries, unsigned e
 }
 
 //------------------------------------------------------------------------
-// getLikelyClass: find class profile data for an IL offset, and return the most likely class
+// getLikelyClasses: find class profile data for an IL offset, and return the most likely classes
 //
 // Arguments:
+//    pLikelyClasses - [OUT] array of likely classes sorted by likelihood (descending), must be at
+//                     least of maxLikelyClasses long (next argument)
+//    maxLikelyClasses - limit for likely classes to output
 //    schema - profile schema
 //    countSchemaItems - number of items in the schema
 //    pInstrumentationData - associated data
 //    ilOffset - il offset of the callvirt
-//    pLikelihood - [OUT] likelihood of observing that entry [0...100]
-//    pNumberOfClasses - [OUT] estimated number of classes seen at runtime
 //
 // Returns:
-//    Class handle for the most likely class, or nullptr
+//    Estimated number of classes seen at runtime
```

Comment:
We should also clearly document:
* if estimated classes is less than max, what is the contents of tail portion of the likely classes array
* if an entry within the [0... numclasses-1]  can have a null class handle